db/migrations/20200528005020_access_codes.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('access_codes', function (table) {
+    table.string('id', 6).primary();
+    table.integer('case_id').unsigned().unique().references('cases.id').onDelete('CASCADE');
```

Comment:
Touched on this in an earlier comment but I think we need to include a field to persist the actual value of the access code.

Reply:
`id` is the 6-digit value and primary key, possibly confusing given the conversation happening in Slack